broad-singlecellportal-bistlin.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "singlecellportal",
+  "owner": "broad-singlecellportal-bistlin",
+  "intent": "Developer MongoDB VM for Single Cell Portal",
+  "production": false,
+  "stability": "volatile",
+  "environment": "development",
+  "project": "broad-singlecellportal-bistlin",
+  "single_cell_db": "single_cell_portal_development",
```

Comment:
I don't think this has any effect.

Reply:
Good catch!  I ended up using `SCP_MONGODB_DB_NAME`, lower in this file, and overlooked cleaning this up.  Fixed in 1518e06.